server/audit_math/supersimple.py

```diff
@@ -139,7 +139,13 @@ def compute_discrepancies(
                 if e:
                     # we found a discrepancy!
                     found = True
-                e_weighted = e / V_wl
+
+                if not V_wl:
+                    # In this case the error is undefined
+                    e_weighted = -1.0
```

Comment:
can you explain this case? i dont understand it. when would `V_wl` be 0 and why do you then set `e_weighted` to -1.0?

Reply:
If V_wl is 0, the race was tied, meaning that the denominator of the weighted error is 0 and the weighted error approaches infinity. I'm using -1 here to code that this has occurred, so later on we can gracefully handle some of the math. I suppose I should pick a better value like `inf`.

Reply:
ah that makes sense

Reply:
agreed, inf would be better